conforming/README.md

```diff
@@ -0,0 +1,163 @@
+
+### Security
+
+The conforming agent will be configurable to support the following http/s based security mechanisms:
+- Basic Authentication
+- Oauth2
+
+## Deployment
+
+### Compile, Test & Package
+
+```console
+mvn package
+```
+
+This will generate 
+- a [plugin jar](target/original-conforming-agent-1.9.5-SNAPSHOT.jar) containing all necessary components to be dropped into a Jakarta-Compatible Web Server.
+- a [standalone jar](target/conforming-agent-1.9.5-SNAPSHOT.jar) including the Jakarta-Reference Implementation (Glassfish).
+
+### Run Locally
+
+The [standalone jar](target/conforming-agent-1.9.5-SNAPSHOT.jar) may be started as follows
+
+```console
+java -cp target/conforming-agent-1.9.5-SNAPSHOT.jar org.eclipse.tractusx.agents.conforming.Bootstrap"
+```
+
+### Containerizing
+
+You could either call
+
+```console
+mvn install -Pwith-docker-image
+```
+
+or invoke the following docker command after a successful package run
+
+```console
+docker build -t ghcr.io/eclipse-tractusx/knowledge-agents/conforming-agent:1.9.5-SNAPSHOT -f src/main/docker/Dockerfile .
+```
+
+This will create a docker image based on a minimal java environment for running the Glassfish-based standalone jar.
+
+To run the docker image, you could invoke this command
+
+```console
+docker run -p 8080:8080 \
+  ghcr.io/eclipse-tractusx/knowledge-agents/conforming-agent:1.9.5-SNAPSHOT
+````
+
+Afterwards, you should be able to access the [local SparQL endpoint](http://localhost:8080/) via
+the browser or by directly invoking a query
+
+```console
+curl --location --request POST 'http://localhost:8080/bind' \
+--header 'Content-Type: application/sparql-query' \
+--header 'Accept: application/json' \
+--data-raw 'PREFIX : <http://example.org/voc#>
+
+SELECT ?x
+WHERE {
+   ?x a :Professor .
+}'
+```
+
+You may manipulate any of the following environment variables to configure the image behaviour.
+Note that there is no builtin security (ssl/auth) for the exposed endpoints.
+This must be provided by hiding them in an appropriate service network layer.
+
+| ENVIRONMENT VARIABLE        | Required  | Example                                                                | Description                          | List |
+|---	                        |---	      |---	                                                                   |---                                   | ---  |
+| JAVA_TOOL_OPTIONS           |           | -agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:8090   | JMV (Debugging option)               | X    | 
+
+### Helm Chart
+
+A helm chart for deploying the remoting agent can be found under [this folder](../charts/remoting-agent).
+
+It can be added to your umbrella chart.yaml by the following snippet
+
+```console
+dependencies:
+  - name: conforming-agent
+    repository: https://catenax-ng.github.io/product-knowledge/infrastructure
```

Comment:
After the initial contribution you should have GH workflows in place which will release your application and other artifacts like Helm chart.

Do not link to catenax-ng repositories/GH Org.

Reply:
switched to the eclipse tractusx helm repo (dev)